app/src/test/java/com/example/mylittlelibrary/ExampleUnitTest.kt

```diff
@@ -6,7 +6,6 @@ import org.junit.Assert.*
 
 /**
  * Example local unit test, which will execute on the development machine (host).
- *
```

Comment:
Empty line removed from multi-line comment to better match the example on kotlinlang.org. https://kotlinlang.org/docs/coding-conventions.html#documentation-comments